### PR TITLE
Update ADMIN.md

### PR DESCRIPTION
**Pull request by alexAubin**

### CODE REVIEW

**Comment by SimonLefort** [comment]
When I try to prepare the environment : 
```bash
app=borg
source /var/www/$app/venv/bin/activate
(...)
```

I have an error : 
```bash
$ source /var/www/$app/venv/bin/activate
bash: /var/www/borg/venv/bin/activate: No such file or directory
```

My solution ( https://github.com/YunoHost-Apps/borg_ynh/pull/167 ) : 
```bash
borg="/var/www/borg/venv/bin/borg"
```

**Comment by alexAubin** [comment]
Hmpf, the lack of `activate` script seems related to using `--upgrade` when creating the venv ...

@Salamandar : do you understand why you added `--upgrade` in https://github.com/YunoHost-Apps/borg_ynh/commit/ea3f6e34920d821bf3309758d9f33cffa9ab3082 ? x_x

**Comment by alexAubin** [comment]
Anyway reading the `activate` script, it basically just prepends `/full/path/to/venv/bin` to the $PATH var so we can also just do this ...

File: doc/ADMIN.md [hunk 1]
```diff
@@ -10,19 +14,59 @@ systemctl start borg
...
+
+The config panel has a "Last backup list" that allow to have quick look at the recently created backup archives.
+
+However, you may want to manually inspect that the backups are indeed made regularly and contain the expected content.
+
+First, prepare the environment with the appropriate borg variables, etc:
+
+```bash
+app=borg
+source /var/www/$app/venv/bin/activate
```
Comment (alexAubin) [inline doc/ADMIN.md:27]:
```suggestion
PATH="/var/www/$app/venv/bin/:$PATH"
```

**Pull request merged**